### DISCUSSION TITLE
how to add database_engine_options option in celery config

### DISCUSSION THREAD

**Discussion by sdhzlzhk** [apache/airflow#19952] [General]
I want to add database_engine_options  option in airflow config, but not found this option

**Comment by potiuk** [comment]
http://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#sql-alchemy-connect-args

**Reply by sdhzlzhk** [reply]
Thank you for your reply. I used CeleryExecutor, the default celery config has not database_engine_options option. The sql-alchemy-connect-args are not active for celery executor, only the celery_config_options. The source code is as follows:
![image](https://user-images.githubusercontent.com/7436812/144532088-f8e8358e-d2f4-42e0-af23-2932c79d682e.png)
https://github.com/apache/airflow/blob/main/airflow/executors/celery_executor.py
but default celery config occursed Mysql error(4031, 'The client was disconnected by the server because of inactivity')
![image](https://user-images.githubusercontent.com/7436812/144532384-e5f4e109-c220-46b8-9ad2-f03139ea5fc6.png)

**Reply by potiuk** [reply]
Ok. I see.  Why don't you add it as PR then ? Seems you already found a place where to add it ?
